crates/node_binding/src/tokio_runtime.rs

```diff
@@ -0,0 +1,18 @@
+use once_cell::sync::Lazy;
+
+/// TODO(hyf0): we might need a way to cleanup this resource
+static TOKIO_RT: Lazy<tokio::runtime::Runtime> = Lazy::new(init_tokio_runtime);
+
+fn init_tokio_runtime() -> tokio::runtime::Runtime {
+  //
+  tokio::runtime::Builder::new_multi_thread()
+    // See https://github.com/web-infra-dev/rspack/pull/183
+    // 6mb+
+    .thread_stack_size(6777216)
```

Comment:
What are the consequences of having a large stack size?

Reply:
It should affect our memory usage. The default stack size of tokio is 2MiB.

If the stack size limit for Tokio worker threads is set too high, it can lead to several consequences:

1. Memory usage: Each worker thread has its own stack, and a large stack size limit can consume a significant amount of memory. This can be a problem in memory-constrained environments and can result in slower performance and even crashes if the system runs out of memory.

2. Resource allocation: A large stack size limit can impact resource allocation, as the operating system needs to allocate memory for each worker thread's stack. This can lead to less memory being available for other processes, which can affect overall system performance.

3. Context switching: A large stack size limit can increase the time it takes to switch between threads, as more memory needs to be saved and restored. This can impact the responsiveness of the system and can result in lower throughput.

4. Scalability: A large stack size limit can limit the scalability of the system, as each worker thread consumes a significant amount of memory. This can result in a smaller number of worker threads being available, which can limit the amount of parallelism in the system.